src/ui/blockUi-editor.js

```diff
@@ -17,8 +17,35 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
     fluid.defaults("sjrk.storyTelling.blockUi.editor", {
         gradeNames: ["sjrk.storyTelling.blockUi"],
         selectors: {
+            moveBlockDownButton: ".sjrkc-st-reorderer-move-down",
+            moveBlockUpButton: ".sjrkc-st-reorderer-move-up",
             selectedCheckbox: ".sjrkc-st-block-selection-checkbox"
         },
+        events: {
+            onReadyToBind: null,
+            onBlockControlsBound: null,
+            onMoveBlockDown: null,
```

Comment:
This is an excellent suggestion, which I've implemented for the most part.

I'm hesitant to remove the final `reorderBlock` invoker, since leaving it in means there's another spot that a programmer can reach in to change things without having to know the details of `sjrk.storyTelling.ui.storyEditor.reorderBlock`'s arguments. This really only saves having to pass one more argument (you'd need a reference to the `reorderer` in order to call the invoker anyway), so it won't be a massive difference, which is why I'm on the fence.